uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/LeaveRequestCalendarFeedConfig.php

```diff
@@ -48,7 +48,11 @@ public function browse() {
 
     CRM_Core_Resources::singleton()->addStyleFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'css/leaveandabsence.css');
     CRM_Core_Resources::singleton()->addScriptFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'js/crm/hrleaveandabsences.js', CRM_Core_Resources::DEFAULT_WEIGHT, 'html-header');
+    CRM_Core_Resources::singleton()->addScriptFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'js/angular/dist/calendar-feeds.list.min.js', 1000);
```

Comment:
Why do we hardcode the weight here? If we need to set the weight opposing to `CRM_Core_Resources::DEFAULT_WEIGHT`, we may want to do smth like this:

```CRM_Core_Resources::DEFAULT_WEIGHT + 1```
or
```CRM_Core_Resources::DEFAULT_WEIGHT - 1```

depending on what do you need. That will be better for readability and will justify the choice of the weight.

Reply:
i think the important thing is that the weight of this file is greater than the weight of `reqangular.min.js`, rather than the default weight

Reply:
1001 was decided since reqangular defines a 1000 value.